internal/config/config.go

```diff
@@ -125,6 +125,13 @@ type Library struct {
 	// overrides Default.Transport.
 	Transport string `yaml:"transport,omitempty"`
 
+	// DescriptionOverride overrides the library description.
```

Comment:
I would split this into a separate PR since it seems unrelated to repometadata

Reply:
It's used by repometadata. (The original version used a Python-specific description override; this is now using a general one.)

I could:

- Leave it as-is, with that extra context
- Change this PR to not use an override at all, and then introduce a second PR which moves it from being Rust-specific to being general, and use it in repometadata at that point
- Introduce a new PR which *just* moves it from being Rest-specific to being general, and make this the "second PR".

Thoughts?

Reply:
I would do 2 or 3